tools.yml

```diff
@@ -2558,3 +2561,21 @@ tools:
     mem: 32
   wig_to_bigWig:
     mem: 10
+destinations:
+  default:
+    params:
+      tpv_cores: '{cores}'
+      tpv_gpus: '{gpus}'
+      tpv_mem: '{mem}'
+    abstract: true
+  tpvdb_slurm:
+    context:
+      time: ''
+      partition: ''
+      account: ''
+    params:
+      native_specification: --nodes=1 --ntasks={cores} --mem={round(mem*1024)} {'--time='
```

Comment:
given that this is multiline, it could be clearer to use one argument per line
--nodes=1
--ntasks={cores}
etc

Reply:
This was because of`tpv format`. I've tried an alternative formatting that at least preserves everything in one, very lengthy line.

Reply:
Fair enough. I hadn't thought about tpv format. This looks good as is